core/arch/arm/kernel/secure_partition.c

```diff
@@ -525,6 +526,92 @@ static TEE_Result handle_fdt_dev_regions(struct sp_ctx *ctx, void *fdt)
 	return TEE_SUCCESS;
 }
 
+#ifdef CFG_CORE_TPM_EVENT_LOG
+static TEE_Result handle_tpm_event_log(void *fdt, struct sp_session *s)
+{
+	int node = 0;
```

Comment:
An upside down christmas tree isn't a requirement, but in cases like this it tends to be a bit easier on the eyes.

Reply:
Done